binaryaudit/dnf.py

```diff
@@ -135,9 +134,13 @@ def generate_abidiffs(key, source_dir, new_json_file, old_json_file, output_dir,
         for arg in cmd_supporting_args:
             command_list.append(arg)
         with open("output_file", "w") as output_file:
-            start_time = time.time()
+            start_time = time.monotonic()
+            print(start_time)
             abipkgdiff, abipkgdiff_exit_code = run.run_command(command_list, None, output_file)
-            exec_time = time.time()-start_time
+            end_time = time.monotonic()
+            print(end_time)
+            exec_time = (end_time-start_time)*1000000
+            print(exec_time)
```

Comment:
This change looks ok, just should please use the output routines from `util.py`. Maybe start/end time itself is not that interesting for the output, just the resulting diff.

But not sure, what causes the surrounding diff yet, as it all has landed in the previous patch. The only part relevant is what i see in your feature branch https://github.com/athvu/python-binaryaudit/commit/ec0959ba53ddc3b86f4eb856a6e1e8c1fc74bdf5 . Could you reset the feature branch to upstream main and reapply only the relevant part please?

Thanks